docs/admin/troubleshooting_connection.rst

```diff
@@ -203,6 +203,39 @@ at the same time. Start ``sd-proxy``, which will bring up ``sd-whonix``.
 
 If this does not resolve the issue, please contact us for assistance.
 
+Customizing Synchronization Timeouts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The SecureDrop Workstation Client application performs a synchronization on
+launch, which may time out in situations where there are a large number of 
+sources on the server. To the extent possible, we recommend regularly removing
+sources that are no longer actively engaging with you, or any sources that
+appear to be spam.
+
+If you are able to login to the Client application, but sources are not
+appearing, you may be reaching a timeout for the synchronization process.
+To temporarily increase the timeout, and give the system more time to
+finish synchronizing with the server, you can perform the following steps:
+
+1. Log into the Qubes workstation
+2. Start a system Terminal in ``dom0`` by going to the Qubes Menu, then choose
+   *Terminal Emulator*
+3. Run the following commands::
+   
+       qvm-service --enable sd-app SDEXTENDEDTIMEOUT_600
+       qvm-shutdown sd-app && sleep 5 && qvm-start sd-app
+   
+4. Start the SecureDrop Client application, like normal
+
+These instructions will change the sync timeout to 10 minutes. If the sync
+fails to complete in that time, repeat the instructions, replacing the "600"
+with a larger value. We recommend increasing the number in increments
+of 100, and checking after each change to find the best value.
+
+Once the synchronization has completed and the sources have loaded, we advise
+going through and removing any unnecessary sources.
```

Comment:
Do we want people to remove the timeout after this? Or just leave it in place?

Reply:
My intuition is that folks who discover they need this should keep it for the time being (i.e., until synchronization itself is more performant).

Reply:
I agree with @cfm, until those performance changes land I imagine this setting would be semi-permanent (with the possibility that folks might need to _increase_ the value as time goes on, assuming more sources are stored on the server).

Reply:
OK!